docs/source/qvm.rst

```diff
@@ -1,7 +1,13 @@
 .. _qvm:
 
+The Quantum Computer
+====================
+
+fdsfds
```

Comment:
?

Reply:
Ah, how did that get in there :innocent:

Reply:
@mpharrigan thanks for catching, it's removed now